client/src/modules/contact.module.css

```diff
@@ -0,0 +1,74 @@
+@import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Baloo+2:wght@400;600;800&family=PT+Serif+Caption&display=swap');
+
+.root {
+    display: inline;
+    height: 100%;
+}
+
+.title {
+    font-family: 'Baloo 2', cursive;
+    text-transform: uppercase;
+    width: 926px;

```

Comment:
these `width: 926px` will be fixed until the screen size hits `768px`, so until then it would be wider than the actual screen size

Reply:
also no margin when the screen size shrinks?

Reply:
also the width of the `title` and `subtitle` is not responsive

Reply:
Done!